fl4health/clients/basic_client.py

```diff
@@ -420,19 +424,24 @@ def _handle_logging(
             current_epoch (Optional[int]): The current epoch of local training.
             logging_mode (LoggingMode): The logging mode (Training, Validation, or Testing).
         """
+        log(INFO, "")  # An empty log line for aesthetics
+
         initial_log_str = f"Current FL Round: {str(current_round)}\t" if current_round is not None else ""
         initial_log_str += f"Current Epoch: {str(current_epoch)}" if current_epoch is not None else ""
+        initial_log_str += self.add_to_initial_log_str  # Add user defined info
         if initial_log_str != "":
             log(INFO, initial_log_str)
+            self.add_to_initial_log_str = ""  # Reset variable
 
-        metric_string = "\t".join([f"{key}: {str(val)}" for key, val in metrics_dict.items()])
-        loss_string = "\t".join([f"{key}: {str(val)}" for key, val in loss_dict.items()])
-
+        # Log loss/losses
         metric_prefix = logging_mode.value
-        log(
-            INFO,
-            f"Client {metric_prefix} Losses: {loss_string} \n" f"Client {metric_prefix} Metrics: {metric_string}",
-        )
+        log(INFO, f"Client {metric_prefix} Losses:")
+        [log(INFO, f"\t {key}: {str(val)}") for key, val in loss_dict.items()]
```

Comment:
Interesting. I didn't know you could actually wrap a log this way.

Reply:
neither did I, just seemed like something that should work so I tried it and it did. My editor wraps at 80 characters so I like to minimize lines as much as possible if they'll fit since my files are already longer bc of the word wrapping